package.json

```diff
@@ -3,6 +3,12 @@
     "react-native-get-random-values": "1.4.0",
     "react-native-reanimated": "2.4.1",
     "react-native-safe-area-context": "3.2.0",
+    "react-native-screens": "2.9.0",
+    "react-native-svg": "9.13.6",
+    "react-native-webview": "11.6.5",
+    "@react-native-masked-view/masked-view": "0.2.6",
```

Comment:
Similar to other dependencies, the current version of this package differs from the one referenced in the tarball. The right version is `0.1.10` ([reference](https://github.com/wordpress-mobile/react-native-masked-view/pull/3/files#diff-7ae45ad102eab3b6d7e7896acd08c427a9b25b346470d7bc6507b6481575d519L8-R8)).